modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -162,6 +171,14 @@ export default {
       }
 
     },
+    getCellStyles(item) {
+      if (this.template && item.dimensions) {
```

Comment:
Let's take `this.template` out of it and arrange things so that `item.dimensions` just doesn't exist except when we want this kind of treatment. Then it becomes a legitimate general purpose feature of the core, and not an intrusion of a pro module into the core.